.github/workflows/build_test.yml

```diff
@@ -24,122 +24,164 @@ jobs:
     name: Check linting
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v5
         with:
           python-version: '3.8'
-      - name: Display Python version
-        run: python -c "import sys; import os; print(\"\n\".join(os.environ[\"PATH\"].split(os.pathsep))); print(sys.version); print(sys.executable);"
-      - name: Upgrade setuptools, pip and wheel
-        run: python -m pip install -U setuptools pip wheel
-      - name: Install tox
-        run: python -m pip install tox
+      - name: Upgrade and install tools
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U hatch
+          python -m hatch env create default
+          python -m hatch env find
```

Comment:
Is this needed?

Reply:
The find option can be removed.
I'd like to separate env creation from tests to make output clean.